src/openms/source/KERNEL/MSChromatogram.cpp

```diff
@@ -44,9 +44,9 @@ std::ostream& OpenMS::operator<<(std::ostream& os, const MSChromatogram& chrom)
   os << static_cast<const ChromatogramSettings&>(chrom);
 
   //data list
-  for (MSChromatogram::ConstIterator it = chrom.begin(); it != chrom.end(); ++it)
+  for (const ChromatogramPeak& it : chrom)
```

Comment:
Same here… please check variable names. Should not be it if they are not an iterator